### ISSUE TITLE
Indexes missing

### ISSUE THREAD

**Issue by fernandoval** [apilayer/marketstack#7]
Where is the indexes?

There is no endpoint to get index values like:

-    NASDAQ Composite (^IXIC)
-    S&P 500 (^GSPC)
-    Dow Jones Industrial Average (^DJI)
-    IBOVESPA (^BVSP)

**Comment by bvermeulen** [comment]
Same question, is there a plan to implement indexes?

**Comment by ChadRockwell** [comment]
Adding indexes would be fantastic.  In fact, it's the main reason I'm not using marketstack.

**Comment by orshe4** [comment]
I invite you to contribute my library: https://github.com/orshe4/marketstack
It supports receiving a full response from the API by sending several requests in parallel(with different offsets) and combining them

It also published as a NuGet package

**Comment by yongkangzhao** [comment]
I'm also looking for CBOE S&P 500 3-Month Volatility Index, since we got VIX.INDX, it would be nice to have the other one too.
